src/pages/user/Login.page.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { Button } from "@components";
+import { useAuth } from "@providers";
+
+const UserLoginPage = () => {
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [error, setError] = useState<null | unknown>(null);
+
+    const authProvider = useAuth();
+    const navigate = useNavigate();
+
+    const login = async (email: string, password: string) => {
+        try {
+            await authProvider.login(email, password);
+            navigate("/user", { replace: true });
+        } catch (error) {
+            setError(error);
+        }
```

Comment:
Updated the auth provider in my PR #21 to not throw error on login, but if this is a pattern we would like to use then I can update the PR before it gets merged.

Ideally, I would want to use the notification system to let the user know there was an error upon login, which can be handled by the provider's `login()` method and the function can be reused throughout the app without having to worry handling errors specific to login or any auth related procedures.